__tests__/core/model.test.ts

```diff
@@ -226,9 +227,48 @@ describe("Model instantiation", () => {
           id: "chakri",
           name: "Subramanya Chakravarthy"
         })
-      ).toThrow("[mobx-state-tree] name property is declared twice")
+      ).toThrow("[mobx-state-tree] 'name' is a property and cannot be declared as a view")
```

Comment:
**praise:** I like that this is even more specific now. I think that's an improved developer experience.